extras/indexing/src/main/java/mvm/rya/indexing/accumulo/ConfigUtils.java

```diff
@@ -165,42 +163,24 @@ public static boolean createTableIfNotExists(final Configuration conf, final Str
         return false;
     }
 
-    private static String getIndexTableName(final Configuration conf, final String indexTableNameConf, final String altSuffix){
-        String value = conf.get(indexTableNameConf);
-        if (value == null){
-            final String defaultTableName = conf.get(RdfCloudTripleStoreConfiguration.CONF_TBL_PREFIX);
-            Validate.notNull(defaultTableName, indexTableNameConf + " not set and " + RdfCloudTripleStoreConfiguration.CONF_TBL_PREFIX + " not set.  Cannot generate table name.");
-            value = conf.get(RdfCloudTripleStoreConfiguration.CONF_TBL_PREFIX) + altSuffix;
-        }
-        return value;
-    }
-
-    public static String getFreeTextDocTablename(final Configuration conf) {
-        return getIndexTableName(conf, FREE_TEXT_DOC_TABLENAME, "freetext");
-    }
-
-    public static String getFreeTextTermTablename(final Configuration conf) {
-        return getIndexTableName(conf, FREE_TEXT_TERM_TABLENAME, "freetext_term");
+    /**
+     * Lookup the table name prefix in the conf and throw an error if it is null.
+     * TODO get table prefix from RyaDetails -- the Rya instance name
```

Comment:
I feel like getting info from the rya details should happen within RyaSailFactory and not ConfigUtils, so I think you're safe to remove this TODO.